test/core/transport/metadata_map_test.cc

```diff
@@ -97,6 +104,14 @@ TEST(MetadataMapTest, TimeoutEncodeTest) {
   EXPECT_EQ(encoder.output(), "grpc-timeout: deadline=1234\n");
 }
 
+TEST(MetadataMapTest, NonEncodableTrait) {
```

Comment:
Maybe also test that this metadata trait is not processed by `Encode()`?

Reply:
done